frontend/lib/pages/hp-action-harassment.tsx

```diff
@@ -52,25 +52,24 @@ export const HarassmentAllegations1 = stepBuilder.createStep(props => ({
   renderIntro: renderAllegationsIntro,
   renderForm: ctx => <AllegationsFieldset>
     <CheckboxFormField {...ctx.fieldPropsFor('allegForce')}>
-      Used force or threatened to use force
+      used force or threatened to use force
     </CheckboxFormField>
     <CheckboxFormField {...ctx.fieldPropsFor('allegMisleadingInfo')}>
-      knowingly provided false or misleading information on the current occupancy, or rent stabilization status of a unit on any application or construction document for a permit for work to be performed in said building
+      lied about occupancy or rent status of your apartment
```

Comment:
Should this actually say "lied about the occupancy or rent status of your apartment"?  (I added the word "the")

Reply:
yes, that's a good catch!